kustomize/auth/patches.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: phoenix
+spec:
+  template:
+    spec:
+      containers:
+      - name: phoenix
+        env:
+        - name: PHOENIX_ENABLE_AUTH
+          value: "true"
+        # When enabling Auth, you must provide a secret
+        # In this example, first set a Kubernetes Secret with:
+        # kubectl create secret generic phoenix-secret --from-literal=secret-key='[YOUR-SECRET]'
+        - name: PHOENIX_SECRET
+          valueFrom:
+            secretKeyRef:
+              name: phoenix-secret
+              key: secret-key
```

Comment:
Maybe we default this to just a long string but leave the note? Otherwise it will break... It's what I did with railway... I know it's sorta risky but it at least gets people started (similar to having postgres password)